queue/src/fiskaltrust.Middleware.Queue.AzureTableStorage/PosBootstrapper.cs

```diff
@@ -16,13 +16,18 @@ public class PosBootstrapper : IMiddlewareBootstrapper
         public void ConfigureServices(IServiceCollection serviceCollection)
         {
             var logger = serviceCollection.BuildServiceProvider().GetRequiredService<ILogger<IMiddlewareBootstrapper>>();
-            
+
             var storageConfiguration = AzureTableStorageConfiguration.FromConfigurationDictionary(Configuration);
             serviceCollection.AddSingleton(sp => storageConfiguration);
-            
+
             var storageBootStrapper = new AzureTableStorageBootstrapper(Id, Configuration, storageConfiguration, logger);
             storageBootStrapper.ConfigureStorageServices(serviceCollection);
 
+            if (Configuration.ContainsKey("assemblytype"))
```

Comment:
Added this change since it killed the CloudCashBox for at least Italy. We should add a few more tests to do some basic initialization for different Queues to ensure that we can execute